docs/source/guides/converting-to-openapi.rst

```diff
@@ -196,6 +196,27 @@ openapi.use.xml (boolean)
     Enables converting Smithy XML traits to OpenAPI XML properties. (this
     feature is not yet implemented).
 
+openapi.use.convertedExternalDocs ([string])
+    Limits the source of converted "externalDocs" fields to the specified set
+    of names in an :ref:`externaldocumentation-trait`. By default, this is a
+    set of the following values: "Homepage", "Api Reference", "User Guide",
```

Comment:
1. Can we add "Developer Guide" to this list?
2. Why Api and not API? Since these are arbitrary and meant to be displayed directly, I think we should probably use API.
3. Is this list case-insensitive? I think it *probably* should be. Thoughts?
4. Should probably not say "set of names", but "priority ordered list of names"